pkg/ui/templates/bucket.html

```diff
@@ -0,0 +1,55 @@
+{{define "head"}}
+    <meta http-equiv="refresh" content="300"/>
+
+    <link type="text/css" rel="stylesheet" href="{{ pathPrefix }}/static/vendor/rickshaw/rickshaw.min.css?v={{ buildVersion }}">
+    <link type="text/css" rel="stylesheet" href="{{ pathPrefix }}/static/vendor/eonasdan-bootstrap-datetimepicker/bootstrap-datetimepicker.min.css?v={{ buildVersion }}">
+
+    <script src="{{ pathPrefix }}/static/vendor/rickshaw/vendor/d3.v3.js?v={{ buildVersion }}"></script>
+    <script src="{{ pathPrefix }}/static/vendor/rickshaw/vendor/d3.layout.min.js?v={{ buildVersion }}"></script>
+    <script src="{{ pathPrefix }}/static/vendor/rickshaw/rickshaw.min.js?v={{ buildVersion }}"></script>
+    <script src="{{ pathPrefix }}/static/vendor/moment/moment.min.js?v={{ buildVersion }}"></script>
+    <script src="{{ pathPrefix }}/static/vendor/moment/moment-timezone-with-data.min.js?v={{ buildVersion }}"></script>
+    <script src="{{ pathPrefix }}/static/vendor/eonasdan-bootstrap-datetimepicker/bootstrap-datetimepicker.min.js?v={{ buildVersion }}"></script>
+    <script src="{{ pathPrefix }}/static/vendor/bootstrap3-typeahead/bootstrap3-typeahead.min.js?v={{ buildVersion }}"></script>
+    <script src="{{ pathPrefix }}/static/vendor/fuzzy/fuzzy.js?v={{ buildVersion }}"></script>
+
+    <script src="{{ pathPrefix }}/static/vendor/mustache/mustache.min.js?v={{ buildVersion }}"></script>
+    <script src="{{ pathPrefix }}/static/vendor/js/jquery.selection.js?v={{ buildVersion }}"></script>
+    <!-- <script src="{{ pathPrefix }}/static/vendor/js/jquery.hotkeys.js?v={{ buildVersion }}"></script> -->
+
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="{{ pathPrefix }}/static/js/bucket.js?v={{ buildVersion }}"></script>
+
+    <script id="graph_template" type="text/x-handlebars-template"></script>
+
+    <link type="text/css" rel="stylesheet" href="{{ pathPrefix }}/static/css/graph.css?v={{ buildVersion }}">
+{{end}}
+
+{{define "content"}}
+    <script type="text/javascript">
+     var thanos = {
+         label: {{.Label}},
+         err: {{.Err}},
+         refreshedAt: {{.RefreshedAt}},
+         blocks: {{.Blocks}}
+     };
+    </script>
+
+    <div id="err" class="container" style="display: none;">
+        <div class="alert alert-warning" role="alert"></div>
+    </div>
+
+    <div class="container-fluid" id="Compactions" style="height: 100%; width: 100%;"></div>
+
+    <div id="legend" class="container-fluid" style="display: none; padding-top: 50px;">
```

Comment:
Can we add comment with Github issue to fix this? (as to remove legend and make it more clear - do not require user to look what number means) (: Let's create that issue if not there yet!

Reply:
@bwplotka Can track progress here? https://github.com/improbable-eng/thanos/issues/1246#issuecomment-506681398

Reply:
I would say it would be nice to have new issue to be clear, but up to you (:

Reply:
Let me know what's the plan as it's the last item before merge. All rest is good :+1:

Reply:
ping, let's add TODO if we can